chain/docker-config/single-validator/run.sh

```diff
@@ -13,4 +13,4 @@ mv config_tmp.toml config.toml
 cd /zoracle/
 
 mkdir ~/.banddb
-bandd start --rpc.laddr tcp://0.0.0.0:26657 --with-db sqlite3:/root/.banddb/main.db
+bandd start --rpc.laddr tcp://0.0.0.0:26657 --with-db "postgres: host=172.18.0.88 port=5432 user=postgres dbname=postgres password=postgrespassword sslmode=disable"
```

Comment:
Let's add `# TODO` to remind us to remove the user & password. Not sure if it's configured to be accessible via localhost only, but it doesn't look secure here.

Reply:
This docker thing is primarily for testing.